client/state/stats/lists/utils.js

```diff
@@ -372,5 +372,41 @@ export const normalizers = {
 				actionMenu: actions.length
 			};
 		} );
+	},
+
+	/*
+	 * Returns a normalized statsSearchTerms array, ready for use in stats-module
+	 *
+	 * @param  {Object} data   Stats data
+	 * @param  {Object} query  Stats query
+	 * @return {Array}         Parsed data array
+	 */
+	statsSearchTerms( data, query ) {
+		if ( ! data || ! query.period || ! query.date ) {
+			return [];
+		}
+
+		const { startOf } = rangeOfPeriod( query.period, query.date );
+		const searchTerms = get( data, [ 'days', startOf, 'search_terms' ], [] );
+		const encryptedSearchTerms = get( data, [ 'days', startOf, 'encrypted_search_terms' ], false );
+
+		const result = searchTerms.map( ( day ) => {
+			return {
+				label: day.term,
+				className: 'user-selectable',
+				value: day.views
+			};
+		} );
+
+		if ( encryptedSearchTerms ) {
+			result.push( {
+				label: translate( 'Unknown Search Terms' ),
```

Comment:
I'm fine with this - to me it is quite similar to how we localize dates in the normalizer too 👍